src/common/com/intellij/plugins/haxe/ide/HaxeFindUsagesHandler.java

```diff
@@ -135,7 +153,45 @@ public boolean processInReadAction(PsiReference reference) {
   @NotNull
   @Override
   public PsiElement[] getSecondaryElements() {
-    // In Java, if the element is a property, return getters and setters that reference it.
-    return super.getSecondaryElements();
+    PsiElement[] secondaryElements = ReadAction.compute(() -> {
+      if (getPsiElement() instanceof HaxeMethodDeclaration) {
+        return tryGetProperty((HaxeMethodDeclaration)getPsiElement());
+      } else if (getPsiElement() instanceof HaxeFieldDeclaration &&
+                 ((HaxeFieldDeclaration)getPsiElement()).getPropertyDeclaration() != null) {
+        return tryGetPropertyAccessMethods((HaxeFieldDeclaration)getPsiElement());
+      }
+      return null;
+    });
+
+    return secondaryElements == null ? super.getSecondaryElements() : secondaryElements;
+  }
+
+  @Nullable
+  private static PsiElement[] tryGetProperty(HaxeMethodDeclaration method) {
+    final HaxeMethodModel methodModel = method.getModel();
+    final HaxeClassModel declaringClass = methodModel.getDeclaringClass();
+    final String methodName = method.getName();
+
+    HaxeMemberModel classField = null;
+
+    if (StringUtils.startsWith(methodName, GETTER_PREFIX)) {
+      classField = declaringClass.getField(methodName.substring(GETTER_PREFIX.length()));
+    } else if (StringUtils.startsWith(methodName, SETTER_PREFIX)) {
+      classField = declaringClass.getField(methodName.substring(GETTER_PREFIX.length()));
```

Comment:
Cut 'n' paste error?  Should be SETTER_PREFIX.

Reply:
It's not a mistake, it was made intentionally, because length of getter_prefix and setter_prefix are the same. But now i see that this could confuse reader.